src/main/java/org/openrewrite/staticanalysis/AnnotateNullableMethods.java

```diff
@@ -43,33 +50,45 @@ public String getDisplayName() {
 
     @Override
     public String getDescription() {
-        return "Add the `@org.jspecify.annotation.Nullable` to non-private methods that may return `null`. " +
-               "This recipe scans for methods that do not already have a `@Nullable` annotation and checks their return " +
-               "statements for potential null values. It also identifies known methods from standard libraries that may " +
-               "return null, such as methods from `Map`, `Queue`, `Deque`, `NavigableSet`, and `Spliterator`. " +
-               "The return of streams, or lambdas are not taken into account.";
+        return "Add `@Nullable` to non-private methods that may return `null`. " +
+                "By default `org.jspecify.annotations.Nullable` is used, but through the `nullableAnnotationClass` option a custom annotation can be provided." +
+                "When providing a custom `nullableAnnotationClass` that annotation should be meta annotated with `@Target(TYPE_USE)`." +
+                "This recipe scans for methods that do not already have a `@Nullable` annotation and checks their return " +
```

Comment:
Would you mind adding a space after each of the end one line dots here? It's often forgotten, but when these strings get concatenated the missing spaces will make the docs hard to read.